sdk/monitor/Azure.Monitor.Query/tests/LogsQueryClientClientLiveTests.cs

```diff
@@ -428,6 +429,68 @@ public async Task ThrowsExceptionWhenBatchQueryNotFound()
             StringAssert.StartsWith("Query with ID '12345' wasn't part of the batch. Please use the return value of the LogsBatchQuery.AddQuery as the 'queryId' argument.", exception.Message);
         }
 
+        [RecordedTest]
+        [TestCase(true)]
+        [TestCase(false)]
+        public async Task CanQueryWithStatistics(bool include)
+        {
+            var client = CreateClient();
+
+            var response = await client.QueryAsync(TestEnvironment.WorkspaceId, _logsTestData.TableAName, options: new LogsQueryOptions()
+            {
+                IncludeStatistics = include
+            });
+
+            if (include)
+            {
+                Assert.Greater(response.Value.Statistics.GetProperty("query").GetProperty("executionTime").GetDouble(), 0);
+            }
+            else
+            {
+                Assert.AreEqual(JsonValueKind.Undefined, response.Value.Statistics.ValueKind);
+            }
+        }
+
+        [RecordedTest]
+        [TestCase(true)]
+        [TestCase(false)]
+        public async Task CanQueryWithStatisticsBatch(bool include)
+        {
+            var client = CreateClient();
+
+            LogsBatchQuery batch = InstrumentClient(client.CreateBatchQuery());
+            var queryId = batch.AddQuery(TestEnvironment.WorkspaceId, _logsTestData.TableAName, options: new LogsQueryOptions()
+            {
+                IncludeStatistics = include
+            });
+            var batchResult = await batch.SubmitAsync();
+            var result = batchResult.Value.GetResult(queryId);
+
+            if (include)
+            {
+                Assert.Greater(result.Statistics.GetProperty("query").GetProperty("executionTime").GetDouble(), 0);
+            }
+            else
+            {
+                Assert.AreEqual(JsonValueKind.Undefined, result.Statistics.ValueKind);
+            }
+        }
+
+        [RecordedTest]
+        public async Task CanSetServiceTimeout()
+        {
+            var client = CreateClient();
+
+            // TODO: How to test this?
```

Comment:
Would we want to test this with a mock transport in a non-recorded test just to validate that we make the request with the timeout value set?